docker/docker-compose.yml

```diff
@@ -106,6 +106,8 @@ services:
       dockerfile: ./docker/nginx.dockerfile
     hostname: "nginx"
     deploy: *default-deploy
+    environment:
+      - BRANDING_PATH=whamcloud-favicons
```

Comment:
Can we change this to `whamcloud`? Eventually we should move the logos out of the binary as well.

Reply:
sure